controllers/core_monitoring_controller.go

```diff
@@ -254,6 +259,16 @@ var conversions = map[int64]unstructuredConverter{
 	//         compactor_blocks_retention_period:
 	//					 seconds: int64 [move from v0 storage.retentionPeriod]
 	//     cortexWorkloads:
+	//   grafana:
+	//     deployment:
+	//       spec:
+	//         template:
+	//           spec:
+	//             containers:
+	//               - env:
+	//                   - name: GF_VAR
+	//                     value: 'true'
+	//                 name: grafana
```

Comment:
@kralicky I had some trouble testing this mostly because the clusters I had created were already on revision 1. Should we create another revision and put these conversions there or should we maintain it as is?

Reply:
yes, sorry, your migrations should be 1->2.

Reply:
Fixed and tested, it is good to merge now